### PR TITLE
Translate ar

### PR DESCRIPTION
**Pull request by MohamedMaher5**

### CODE REVIEW

File: resources/lang/ar.json [hunk 1]
```diff
@@ -401,7 +401,20 @@
...
+    "60 Days": "٦٠ يوم",
+    "90 Days": "٩٠ يوم",
+    "Today": "اليوم",
+    "Month To Date": "شهر من الآن",
+    "Quarter To Date": "۳ شهور من الآن",
+    "Year To Date": "سنة من الآن",
+    "Customize": "تخصيص",
+    "Update :resource: :title": "تحديث :resource: :title",
+    "Update attached :resource: :title": "تحديث المرفق :resource: :title",
+    ":resource Details: :title": ":resource details: :title"
```
Comment (kitbs) [inline resources/lang/ar.json]: The word "Details" has not been translated.

**Comment by kitbs** [comment]
Hi @MohamedMaher5. Thanks for your contribution! Can you please fix the missing translation in the comment?

File: resources/lang/ar.json (hunk 1 above)
Comment (MohamedMaher5) [reply resources/lang/ar.json]: Done

**Comment by kitbs** [comment]
Thank you @MohamedMaher5 !

**Pull request merged**